modules/nf-core/svync/main.nf

```diff
@@ -0,0 +1,54 @@
+process SVYNC {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/svync:0.1.0--h9ee0642_0':
+        'biocontainers/svync:0.1.0--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi)
+    tuple val(meta2), path(config)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def args2  = task.ext.args2 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    if ("$vcf" == "${prefix}.vcf.gz") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+
+    """
+    svync \\
+        $args \\
+        --config $config \\
+        --input $vcf \\
+        | bgzip --threads $task.cpus $args2 > ${prefix}.vcf.gz
```

Comment:
It's a binary compression, maybe this is why tests aren't reproducible use contains instead of always the same lines?

Reply:
That's the weird part, the stub test is failing not the real run :grin: